en/frames_rover/README.md

```diff
@@ -1,11 +1,49 @@
-# Rover Frames (UGVs)
+# Rovers (UGVs)
 
-PX4 provides basic support for Unmanned Ground Vehicles (UGVs). The set of validated configurations can be seen in [Airframes Reference > Rover](../airframes/airframe_reference.md#rover).
+PX4 supports rovers (Unmanned Ground Vehicles - UGVs) with [ackermann and differential](#rover-types) steering.
 
-This section contains build logs/instructions for assembling and configuring a number of UGV frames.
+This section contains build logs/instructions for assembling as well as configuring a number of UGV frames.
+
+![Traxxas Rover Picture](../../assets/airframes/rover/traxxas_stampede_vxl/final_side.jpg)
+
+## Rover Types
+
+PX4 supports rovers with:
+
+- **Differential steering**: direction is controlled by moving the left- and right-side wheels at different speeds.
+   This kind of steering commonly used on a military tank of the wheels left and right. 
+- **Ackermann steering**: direction is controlled by pointing wheels in the direction of travel ([ackermann geometry](https://en.wikipedia.org/wiki/Ackermann_steering_geometry) compensates for the fact that wheels on the inside and outside of the turn move at different rates).
+  This kind of steering is used on most commercial vehicles, including cars, trucks etc.
+
+The supported frames can be seen in [Airframes Reference > Rover](../airframes/airframe_reference.md#rover).
+
+
+## How to Configure a Rover
+
+Using [control allocation](../config/actuators.md), it is straightforward to setup a rover.
+
+For vehicles with Ackermann steering:
+
+1. In the [Airframe](../config/airframe.md) configuration, select the *Generic Ground Vehicle*.
+1. Set the [CA_AIRFRAME](../advanced_config/parameter_reference.md#CA_AIRFRAME) parameter to **Rover (Ackermann)** (5).
+1. Follow the [Actuators document](../config/actuators.md) to map the steering and throttle outputs as displayed.
+
+For vehicles with Differential steering:
+
+1. In the [Airframe](../config/airframe.md) configuration, select either the _Aion Robotics R1 UGV_ or _NXP Cup car: DF Robot GPX_
```

Comment:
@bkueng I created https://github.com/PX4/PX4-Autopilot/issues/19937 because (among other things) there is no generic differential frame. 
- Is the intent to create one?
- Is the approach here OK for now - ie copying these two frames. My assumption is that they will be able to add any additional throttle or whatever as needed.


Further, the assumption of our generic ackerman is one steering motor/servo and one for thrust. However the simulated rovers have multiple motors for steering and so on defined. 
So my question is, how configurable is this? If I add 2 left motors and 2 right motors will all of them be used. What about a 6 motor vehicle? 

It would be good to state our limits here :-)

Reply:
Thanks @bkueng - then I think this is probably OK for now. I'll merge. If you hate it let me know and we can fix as a post process.